.devcontainer/Dockerfile

```diff
@@ -1,13 +1,23 @@
-FROM maven:3.6.3-jdk-11
+FROM maven:3.6.3-jdk-11-slim
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-RUN apt-get update
-RUN apt-get -y install --no-install-recommends apt-utils dialog dnsutils httpie bash-completion curl wget git unzip
-RUN apt-get -y install git procps lsb-release
+#Install Common Tools
+RUN apt-get update && apt-get -y install --no-install-recommends apt-transport-https apt-utils bash-completion curl dialog dnsutils git httpie lsb-release procps unzip wget
```

Comment:
It'd be good to split up this line a bit, so that the packages installed are listed in it's own line for readability.

Reply:
@sivamu , when you split into separate lines they create new layers.  Could possibly do a line continuation for reading clarity, but I think you want to stay with a single RUN command.

Reply:
@jofultz exactly what I was thinking. I pushed line continuations for better readability, but in the same RUN command.

Reply:
Ah yea, I should've worded it better, that's what I was meaning to say :)